### PR TITLE
draft censo2017

### PR DESCRIPTION
**Pull request by pachadotdev**
* [x] I have read the Content Guidelines.
* [x] I have read the Technical Guidelines.
* [x] I used or followed the R Markdown or Markdown template.
* [x] I have followed the Style Guide.
* [x] I created or updated my author metadata with correct folder name.
* [x] I have added relevant tags after browsing existing tags (incuding "community" tag).
* [x] I have added the "tech notes" tag if this is a technote.
* [x] I ran `roblog::ro_lint_md()` on index.md (optional).
* [x] I ran `roblog::ro_check_urls()` on index.md (optional).
* [x] I ran a spell-check on index.md.
* [x] I have added the tags - Software Peer Review, my-packagename.
* [x] I have added the package-version YAML tag.
* [x] I have added acknowledgement of the reviewers' work (with links to reviewers).
* [x] I have added a link to the software peer review thread.
* [x] I ran a spell-check on index.md.

### CODE REVIEW

**Comment by steffilazerte** [comment]
Thanks @pachadotdev! Sorry I've been slow (busy at the useR conference!) but I'll review within the next few days. 

@noamross I'll be reviewing for editorial stuff soon, but as this package was the first to be reviewed in Spanish, and the Spanish review process was an outcome of your discussions during a community call, would you be able to give it a review too? We'll be asking @maurolepore to come in later and review the Spanish translation (to come!) 

Thanks!

File: content/blog/2021-06-20-censo2017.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+
+The help of rOpenSci and its software review was pivotal. This package is the first of rOpenSci’s scientific software to be documented and reviewed in Spanish. Dedicated reviewers, [María Paula Caldas](https://www.mpaulacaldas.com/), [Frans van Dunné](https://www.fransvandunne.com/) and [Melina Vidoni](/author/melina-vidoni/), tested the package and suggested enhancements. Their reviews were very helpful, they gave extremely valuable advice and were really supportive of my work and the development of the package. 
+
+It was a pleasure for me to count on thoughtful reviews from the same people who I reviewed their contributions to [R for Data Science in Spanish](https://es.r4ds.hadley.nz/) long ago in 2019. If the reader sees a potential bias in the review, there isn’t one, as the reviewing process is open, and even my fellow [Mauro Lepore](https://maurolepore.github.io/) voluntary refused the review as we interacted intensively during R4DS translation process and wrote an article, [Datos: datasets translations to promote diversity, equity, and inclusion](https://github.com/openjournals/jose-reviews/issues/99), as a result of that collaboration.
```
Comment (noamross) [inline content/blog/2021-06-20-censo2017.md:237]: I don't think this paragraph is that necessary.  If you remove it, you can move the links to the next paragraph in a sentence that mentions them as other initiatives in Spanish-language R resources.

**Review by noamross — CHANGES_REQUESTED** [review]
Great post, I just had a few small comments.  While I like the narrative arc, I do think it's usually better to have the first paragraph or two give an introduction that summarizes what is to come (e.g., this post is about this package, the problem it solved, how to use it, and the fact that the package and its review process were all in Spanish).

File: content/blog/2021-06-20-censo2017.md [hunk 2]
```diff
@@ -0,0 +1,243 @@
+---
+title: 'censo2017 - The Story Behind the Package'
```
Comment (steffilazerte) [inline content/blog/2021-06-20-censo2017.md:2]: Can we get something about this being the first package to be reviewed in Spanish here?
Perhaps:

- "The Story Behind censo2017, the First rOpenSci Package to be Reviewed in Spanish" or
- "censo2017 - The Story Behind the First rOpenSci Package to be Reviewed in Spanish"

What do you think?

**Review by steffilazerte — COMMENTED** [review]
Hi Pachá! 

This is looking great, I love the thoughtfulness, and the narrative (as Noam put it). I have just a couple of comments/suggestions (13) left to go. 

I'm away next week Monday through Thursday and I don't want to hold up the process. Noam's and my suggestions are fairly minor, so once you address them, go right ahead and work on the Spanish translation when you have time (i.e. don't wait for me, unless you really want my feedback; I'll leave it up to you to ping me if that's the case).

When you've added the Spanish translation, ping @maurolepore for the Spanish review, and @stefaniebutland for a publication date! We're almost there! Thanks again for your hard work on this post. 

* [X] post follows Content Guidelines
* [x] post follows Style Guide
    - Remove \`  ` from all package names (i.e., censo2017 and citesdb)
* [x] title is in Title Case
  - Definitely in title case, but see what you think about suggestions for being a bit more detailed
* [ ] publication date is ok
  - We'll ask @stefaniebutland after @maurolepore has a chance to look at it
* [X] alternative text of images is informative
* [X] Twitter metadata looks ok (paste post preview link in [Twitter card validator](https://cards-dev.twitter.com/validator); might have to click twice on "Preview card")
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)
* [X] YAML package-version included
* [X] YAML subject tags - software peer review, packagename
* [X] acknowledges and links to reviewers
* [x] links to peer review thread
  - I've suggested a couple of places

**Comment by pachadotdev** [comment]
hi
i added all of the comments

File: content/blog/2021-06-20-censo2017.md [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+
+[^2]: For the period June 2018 - December 2019, the Census was available in [DVD format](https://www.ine.cl/prensa/2019/09/16/ine-pone-a-disposici%C3%B3n-la-base-de-microdatos-del-censo-2017) and REDATAM only. Now it's available [online](https://www.ine.cl/estadisticas/sociales/censos-de-poblacion-y-vivienda/poblacion-y-vivienda) in REDATAM, SPSS and CSV formats.
+
+I had to borrow a Windows laptop and get an external DVD reader in order to read those REDATAM files to obtain a few population summaries with specific software for that format. To my surprise, the task started to get more and more challenging, to the point that I wanted to export the data to SQL for easier data extraction.
+
+My goal wasn’t to extract statistical secrets, which is not posible with these datasets, I just wanted to obtain values of population by age intervals for different regions, among other similar aggregations, which is something that dplyr and other tools ease a lot. After being able to convert the datasets and the description of variables to CSV/XML, I found that the effort in doing that justified creating an R package to organize my work.
```
Comment (steffilazerte) [inline content/blog/2021-06-20-censo2017.md:33]:
```suggestion
My goal wasn’t to extract statistical secrets, which is not possible with these datasets, I just wanted to obtain values of population by age intervals for different regions, among other similar aggregations, which is something that dplyr and other tools ease a lot. After being able to convert the datasets and the description of variables to CSV/XML, I found that the effort in doing that justified creating an R package to organize my work.
```

File: content/blog/2021-06-20-censo2017.md [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+
+# rOpenSci community contribution
+
+The help of rOpenSci and its software review was pivotal. Dedicated reviewers, [María Paula Caldas](https://www.mpaulacaldas.com/), [Frans van Dunné](https://www.fransvandunne.com/) and [Melina Vidoni](/author/melina-vidoni/), tested the package and suggested enhancements. Their reviews were very helpful, they gave extremely valuable advice and were really supportive of my work and the development of the package. I had the chance to interact with them in the past, during R4DS translation process, where I co-wrote an article with rOpenSci's editor [Mauro Lepore](/author/mauro-lepore/), [Datos: datasets translations to promote diversity, equity, and inclusion](https://github.com/openjournals/jose-reviews/issues/99), as a result of that collaboration.
```
Comment (steffilazerte) [inline content/blog/2021-06-20-censo2017.md:241]: Link to review issue for your package

```suggestion
The help of rOpenSci and its [software review](https://github.com/ropensci/software-review/issues/414) was pivotal. Dedicated reviewers, [María Paula Caldas](https://www.mpaulacaldas.com/), [Frans van Dunné](https://www.fransvandunne.com/) and [Melina Vidoni](/author/melina-vidoni/), tested the package and suggested enhancements. Their reviews were very helpful, they gave extremely valuable advice and were really supportive of my work and the development of the package. I had the chance to interact with them in the past, during R4DS translation process, where I co-wrote an article with rOpenSci's editor [Mauro Lepore](/author/mauro-lepore/), [Datos: datasets translations to promote diversity, equity, and inclusion](https://github.com/openjournals/jose-reviews/issues/99), as a result of that collaboration.
```

**Review by steffilazerte — COMMENTED** [review]
So speedy :grin: I fixed one typo and added the link to your packages review issue. @noamross what do you think?

**Comment by noamross** [comment]
Looks good to me!

**Comment by steffilazerte** [comment]
Great! @pachadotdev go ahead with the Spanish. I'm away most of next week, so when that's done, ping @maurolepore for review, and @stefaniebutland for a publication date.

**Comment by pachadotdev** [comment]
btw, the docs work now https://docs.ropensci.org/censo2017/
thanks to Jeroen who fixed the internals and now I can install sf on the CI

**Comment by pachadotdev** [comment]
@maurolepore querido, he subido el post en espaniol aparte de la version en inglés

File: content/blog/2021-06-20-censo2017-espaniol.md [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+10 personas p15         10 Técnica Comercial, Industrial/Normalista (Sistema Antiguo)
+11 personas p15         11 Técnico Superior (1-3 Años)                               
+12 personas p15         12 Profesional (4 o Más Años)                                
+13 personas p15         13 Magíster                                                  
+14 personas p15         14 Doctorado                                                 
+15 personas p15         99 Valor Perdido                                             
+16 personas p15         98 No Aplica
+```
+
+Para obtener información detallada de cada municipio/región en relación con las preguntas anteriores, tenemos que pensar en los datos de REDATAM como un árbol, y es necesario unir "zonas" con "viviendas" por ID de zona, luego unir "viviendas" con "hogares" por ID de vivienda, y luego "hogares" con "personas" por ID de hogar. Esto se hace en nada de tiempo, lo que es nada, con el backend DuckDB.
```
Comment (maurolepore) [inline content/blog/2021-06-20-censo2017-espaniol.md:107]: No entiendo el significado de "lo que es nada" pero la oracion tiene sentido sin eso. Creo "rapidamente" es mas preciso que "nada de tiempo". Sugiero:

> Esto se hace rapidamente con el backend DuckDB.

File: content/blog/2021-06-20-censo2017-espaniol.md [hunk 6]
```diff
@@ -0,0 +1,247 @@
...
+    zonaloc_ref_id, vivienda_ref_id), by = "zonaloc_ref_id") %>%
+  inner_join(select(tbl(censo_conectar(), "hogares"), vivienda_ref_id, 
+    hogar_ref_id), by = "vivienda_ref_id") %>%
+  inner_join(select(tbl(censo_conectar(), "personas"), hogar_ref_id,
+    p08, p15, p18), by = "hogar_ref_id") %>%
+  collect()
+```
+
+Se puede obtener una rápida verificación con una simple cuenta. La tabla resultante tiene mucho sentido y refleja estadísticas resumidas de conocimiento común, como que en la capital (región #13) hay unos 7 millones de habitantes.
+
```
Comment (maurolepore) [inline content/blog/2021-06-20-censo2017-espaniol.md:126]: Creo que es mejor elimiar "simple" y "tiene mucho sentido" para no herir a quienes encuentren la cuenta y tabla resultante dificiles de entender.

File: content/blog/2021-06-20-censo2017-espaniol.md [hunk 7]
```diff
@@ -0,0 +1,247 @@
...
+ 9 09      957224
+10 10      828708
+11 11      103158
+12 12      166533
+13 13     7112808
+14 14      384837
+15 15      226068
+```
+
+Para conocer la proporción de hombres (1) y mujeres (2) por región, es muy similar.
```
Comment (maurolepore) [inline content/blog/2021-06-20-censo2017-espaniol.md:151]: Creo que esta oracion se lee mejor escribiendo explicitamente "el codigo" (o el sujeto que corresponda). Sugiero:

> "..., [el codigo] es muy similar."

File: content/blog/2021-06-20-censo2017-espaniol.md [hunk 8]
```diff
@@ -0,0 +1,247 @@
...
+95 percent confidence interval:
+ -0.06407740 -0.06266263
+sample estimates:
+   prop 1    prop 2
+0.4338752 0.4972452
+```
+
+La estimación del intervalo de confianza del 95% de la diferencia entre la proporción de hombres ancianos y no ancianos es de entre -6,4% y -6,3%, lo que está en consonancia con el hecho de que las mujeres tienden a tener una mayor esperanza de vida en Chile (consulta [Statista](https://www.statista.com/statistics/949308/life-expectancy-birth-chile-gender/) para comparar). Lo que quería demostrar aquí es que usando este paquete, es muy fácil pasar datos a R, para poder realizar análisis de regresión u otros análisis estadísticos sobre la población chilena.
+
+# Enlaces con formación y política
```
Comment (maurolepore) [inline content/blog/2021-06-20-censo2017-espaniol.md:229]: Esta seccion parece contaner dos partes -- una conclusion de la utilidad del paquete, y una opinion politica. Necesitas dos encabezados en vez de solo uno?

File: content/blog/2021-06-20-censo2017-espaniol.md [hunk 9]
```diff
@@ -0,0 +1,247 @@
...
+
+La base de datos censales que proporciono dentro del paquete puede ser leída en Python, Java C++, node.js e incluso desde una línea de comandos, y es importante mencionar que es multiplataforma. Para el caso del análisis estadístico aplicado basado en GUI, censo2017 hace muy fácil la exportación de subconjuntos a Microsoft Excel o Google Sheets, sin las desventajas del formato original del censo anteriormente comentado, e incluso a Stata y SPSS gracias al ecosistema de paquetes R.
+
+Las universidades y centros de investigación en América Latina carecen de financiamiento y los gobiernos no tienen una agenda política basada en la evidencia, lo cual es una combinación peligrosa. Chile, al ser parte de América Latina, no es la excepción, y en las últimas décadas, la estabilidad política y el crecimiento impulsado por el comercio han logrado elevar a cientos de millones por sobre de la línea de pobreza. En el país y en la región hay un margen considerable para seguir cosechando los beneficios que ofrece una agenda gubernamental racional basada en la evidencia.
```
Comment (maurolepore) [inline content/blog/2021-06-20-censo2017-espaniol.md:235]: Es este un buen espacio para ofrecer una opinion politica? 

En tal caso, creo que este parrafo necesita informacion mas precisa, por ejemplo, la palabra "carecer" sugiere "cero" pero -- sin saber -- supongo que mas preciso seria "poco" o "menos que XXX anios atras".

File: content/blog/2021-06-20-censo2017-espaniol.md [hunk 10]
```diff
@@ -0,0 +1,247 @@
...
+
+# Contribución de la comunidad rOpenSci
+
+La ayuda de rOpenSci y su [revisión de software](https://github.com/ropensci/software-review/issues/414) fue fundamental. Los revisores, [María Paula Caldas](https://www.mpaulacaldas.com/), [Frans van Dunné](https://www.fransvandunne.com/) y [Melina Vidoni](/autor/melina-vidoni/), probaron el paquete y sugirieron mejoras. Sus revisiones fueron muy útiles, dieron consejos muy valiosos y apoyaron mucho mi trabajo y el desarrollo del paquete. Tuve la oportunidad de interactuar con ellos en el pasado, durante el proceso de traducción de R4DS, donde co-escribí un artículo con el editor de rOpenSci [Mauro Lepore](/autor/mauro-lepore/), [Datos: traducciones de conjuntos de datos para promover la diversidad, la equidad y la inclusión](https://github.com/openjournals/jose-reviews/issues/99), como resultado de esa colaboración.
```
Comment (maurolepore) [inline content/blog/2021-06-20-censo2017-espaniol.md:241]: Sugiero eliminar la comma en "... Frans van Dunne y Melina Vidoni[,] probaron el paquete"

Comment (maurolepore) [reply content/blog/2021-06-20-censo2017-espaniol.md:241]: Sugiero eliminar la siguiente oracion. El haber trabajado antes con los revisores sugiere un conflicto de interes. Y el haber trajabado juntos es un honor para mi pero parece poco relacionado a censo2017. 

> Tuve la oportunidad de interactuar con ellos en el pasado, durante el proceso de traducción de R4DS, donde co-escribí un artículo con el editor de rOpenSci [Mauro Lepore](/autor/mauro-lepore/), [Datos: traducciones de conjuntos de datos para promover la diversidad, la equidad y la inclusión](https://github.com/openjournals/jose-reviews/issues/99), como resultado de esa colaboración.

File: content/blog/2021-06-20-censo2017-espaniol.md [hunk 11]
```diff
@@ -0,0 +1,247 @@
...
+
+Me encantaría ver diferentes "copias" de censo2017 en los países vecinos Perú y Bolivia, y el resto de la región. Crear el primer paquete de rOpenSci totalmente documentado en español no sólo tiene que ver con la inclusión, que obviamente es deseable, sino que también tiene que ver con la productividad, porque en América Latina, los materiales que están documentados en inglés son difíciles de entender y por lo tanto las herramientas no son adoptadas como uno quisiera. Así como la infraestructura de R4DS en español motivó la creación de R4DS en portugués, espero que censo2017 abra la puerta a colaboraciones interesantes y paquetes similares en diferentes idiomas.
```
Comment (maurolepore) [inline content/blog/2021-06-20-censo2017-espaniol.md:245]: Esta oracion parece compleja. Quizas puedas descomponerla en partes mas pequenias?

> Crear el primer paquete de rOpenSci totalmente documentado en español no sólo tiene que ver con la inclusión, que obviamente es deseable, sino que también tiene que ver con la productividad, porque en América Latina, los materiales que están documentados en inglés son difíciles de entender y por lo tanto las herramientas no son adoptadas como uno quisiera.

File: content/blog/2021-06-20-censo2017-espaniol.md [hunk 12]
```diff
@@ -0,0 +1,247 @@
...
+
+Creo que hago lo mejor que puedo para contribuir a iniciativas como rOpenSci porque aquí he encontrado mi lugar y he descubierto a otros que tienen el deseo de pasar de la ignorancia al conocimiento en muchas cosas diferentes, construimos las herramientas para ayudarnos a nosotros mismos y a otros en ese proceso. Mi contribución actual es hacer que los datos, las herramientas y las mejores prácticas sean más descubribles, contribuyendo al mismo tiempo a la infraestructura social.
```
Comment (maurolepore) [inline content/blog/2021-06-20-censo2017-espaniol.md:247]: Sugiero reemplazer "pasar de la ignorancia al conocimiento en" por "aprender".

> ... he descubierto a otros que tienen el deseo de [aprender] muchas cosas diferentes ...

**Review by maurolepore — APPROVED** [review]
Muchas gracias por este articulo! Se lo ve muy bien. Lo apruebo pero por favor considera mis sugerencias.

Que honor que el primer paquete revisado en espaniol sea tuyo! Gracias por tu contribucion.

**Comment by pachadotdev** [comment]
@maurolepore anoche arreglé todo lo que comentabas, cualquier punto de diferencia lo dejé fuera, para mantener el común denominador

**Comment by pachadotdev** [comment]
hi
it's all ready, and I added an example with maps

hola
está todo listo y he agregado un ejemplo con mapas

**Review by pachadotdev — COMMENTED** [review]
> Muchas gracias por este articulo! Se lo ve muy bien. Lo apruebo pero por favor considera mis sugerencias.
> 
> Que honor que el primer paquete revisado en espaniol sea tuyo! Gracias por tu contribucion.

muchas gracias, Mauro!
incluí todos tus comentarios y pasé tijeras... me fui por lo típico chileno y todo lo que no era "de consenso", quedó fuera (es la única forma de que el resultado sea de consenso, creo)
agregué un mapa del hacinamiento, pues creo que meustra un tema socioeconómico importante
había otras opciones como la cantidad de phds por barrio y otros temas que no van más allá de crear algo anecdótico, pero el hacinamiento es un tema muy serio en la gestión local

File: content/blog/2021-06-20-censo2017/index.md [hunk 13]
```diff
@@ -0,0 +1,538 @@
...
+          aes(fill = prop_mean, geometry = geometry)) +
+  geom_sf(data = filter(mapa_comunas, codigo_region == "13"),
+          aes(geometry = geometry), colour = "#2A2B75", fill = NA) +
+  ylim(-33.65, -33.3) +
+  xlim(-70.85, -70.45) +
+  scale_fill_gradientn(colors = rev(colRoz_pal("ngadju")), name = "% dwellings") +
+  labs(title = "Share of dwelling with mean overcrowding in the Metropolitan Region")
+```
+
+{{< imgtxt src="mean-overcrowding-map.png" alt="mean overcrowding map" >}}
```
Comment (steffilazerte) [inline content/blog/2021-06-20-censo2017/index.md:476]: consider using 'figure', for a nicer, bigger picture, and a more detailed alternative text
```suggestion
{{< figure src="mean-overcrowding-map.png" alt="Map of the Metropolitan Region showing polygons representing neighbourhoods and coloured by the degree of overcrowding" >}}
```

File: content/blog/2021-06-20-censo2017-espaniol/index.md [hunk 14]
```diff
@@ -0,0 +1,541 @@
...
+          aes(fill = prop_medio, geometry = geometry)) +
+  geom_sf(data = filter(mapa_comunas, codigo_region == "13"),
+          aes(geometry = geometry), colour = "#2A2B75", fill = NA) +
+  ylim(-33.65, -33.3) +
+  xlim(-70.85, -70.45) +
+  scale_fill_gradientn(colors = rev(colRoz_pal("ngadju")), name = "% viviendas hacinadas") +
+  labs(title = "Porcentaje de viviendas con hacinamiento medio en la Región Metropolitana")
+```
+
+{{< imgtxt src="mapa-hacinamiento-medio.png" alt="mapa de hacinamiento medio" >}}
```
Comment (steffilazerte) [inline content/blog/2021-06-20-censo2017-espaniol/index.md:477]: Consider using "figure" for a nicer, bigger image. Also, see my suggestion in the English version for a more informative alternative text. 

```suggestion
{{< figure src="mapa-hacinamiento-medio.png" alt="mapa de hacinamiento medio" >}}
```

File: content/blog/2021-06-20-censo2017-espaniol/index.md [hunk 15]
```diff
@@ -0,0 +1,541 @@
+---
+title: 'La Historia de censo2017, el Primer Paquete de rOpenSci Revisado en
+  Español'
+author:
+  - Pachá (aka Mauricio Vargas Sepúlveda)
+date: '2021-06-20'
```
Comment (steffilazerte) [inline content/blog/2021-06-20-censo2017-espaniol/index.md:6]: Also change the name of the folder!
```suggestion
date: '2021-07-27'
```

File: content/blog/2021-06-20-censo2017-espaniol/index.md [hunk 16]
```diff
@@ -0,0 +1,541 @@
...
+  - blog
+tags:
+  - open-science
+  - reproducible-research
+  - data-access
+  - data-extraction
+  - geospatial
+  - community
+  - software-peer-review
+  - censo2017
```
Comment (steffilazerte) [inline content/blog/2021-06-20-censo2017-espaniol/index.md:19]:
```suggestion
  - censo2017
  - Spanish
```

File: content/blog/2021-06-20-censo2017-espaniol/index.md [hunk 17]
```diff
@@ -0,0 +1,541 @@
...
+description: >
+  Las tablas del Censo de Chile al instante.
+---
+
```
Comment (steffilazerte) [inline content/blog/2021-06-20-censo2017-espaniol/index.md:23]: Could you translate into Spanish: "See the English version of this blog post" Thanks!

```suggestion
> See the English version of this blog post: 
> [The Story Behind censo2017, the First rOpenSci Package to be Reviewed in Spanish](/blog/2021/07/27/censo2017)
```

File: content/blog/2021-06-20-censo2017/index.md [hunk 18]
```diff
@@ -0,0 +1,538 @@
...
+  - data-access
+  - data-extraction
+  - geospatial
+  - community
+  - software-peer-review
+  - censo2017
+description: >
+  Chilean census tables at a glance.
+---
+
```
Comment (steffilazerte) [inline content/blog/2021-06-20-censo2017/index.md:22]:
```suggestion
> See the Spanish version of this blog post: 
> [La Historia de censo2017, el Primer Paquete de rOpenSci Revisado en Español](/blog/2021/07/27/censo2017-es)
```

**Review by steffilazerte — COMMENTED** [review]
Hooray! We have a publication date of Tuesday, July 27th 2021!

You'll notice that I made two changes to the pull request:
- I moved the md files into their respective folders and renamed them to index.md
- I changed the 'slug' for the Spanish version to 'censo2017-es'.

This way, there'll be two posts, one in English and one in Spanish. This also means that your overcrowding figure is displayed properly (it wasn't visible otherwise). 


Last things to do:

- [ ] look at the final suggestions I've made and see if you want to accept them
- [ ] I've suggested a link at the beginning of each, leading to the other post. Can you translate the text for the Spanish version for me? Thanks!
- [ ] Change all headings from # to ## (sorry! I missed that in the first round, it needs to be ## or the spacing won't be right.)
- [ ] Change the dates in the yaml and in the folders to 2021-07-27

I'll draft a tweet in the next few days, and ask you for feedback, then if you wouldn't mind, could you create a Spanish version so we can make sure we have tweets in English and Spanish?

So close! Thanks for your work on this!

**Comment by pachadotdev** [comment]
> * look at the final suggestions I've made and see if you want to accept them
> 
> *

- [x] look at the final suggestions I've made and see if you want to accept them _accepted with no repairs_
- [x] I've suggested a link at the beginning of each, leading to the other post. Can you translate the text for the Spanish version for me? Thanks! _translated, thanks!_
- [x] Change all headings from # to ## (sorry! I missed that in the first round, it needs to be ## or the spacing won't be right.) _ready_
- [x] Change the dates in the yaml and in the folders to 2021-07-27 _ready_

**Comment by steffilazerte** [comment]
Looks great! Thanks :grin: We're ready to publish on Tuesday, the only last thing I'll ask of you is to review this draft text for a tweet (below), and then translate it into Spanish for me (we'll send it out from the rOpenSci account). Thank you! 🙏🏼 

---
[blog] The Story Behind censo2017, the First rOpenSci Package to be Reviewed in Spanish by @pachadotdev

Accessing Chilean 🇨🇱 census data has never been easier!

https://ropensci.org/blog/2021/06/27/censo2017/

#rstats

(1/2)

---

And perhaps best of all, this package was both written AND reviewed in Spanish to maximize accessibility!

🙏 Thank you to rOpenSci reviewers @&#8203;mapaulacaldas, @&#8203;fransvandunne & @&#8203;melvidoni for their contributions to the first ever rOpenSci Spanish package review!

(2/2)

**Comment by pachadotdev** [comment]
> Looks great! Thanks grin We're ready to publish on Tuesday, the only last thing I'll ask of you is to review this draft text for a tweet (below), and then translate it into Spanish for me (we'll send it out from the rOpenSci account). Thank you! 🙏🏼
> 
> [blog] The Story Behind censo2017, the First rOpenSci Package to be Reviewed in Spanish by @pachadotdev
> 
> Accessing Chilean chile census data has never been easier!
> 
> https://ropensci.org/blog/2021/06/27/censo2017/
> 
> #rstats
> 
> (1/2)
> 
> And perhaps best of all, this package was both written AND reviewed in Spanish to maximize accessibility!
> 
> pray Thank you to rOpenSci reviewers @​mapaulacaldas, @​fransvandunne & @​melvidoni for their contributions to the first ever rOpenSci Spanish package review!
> 
> (2/2)

@steffilazerte Hi, here are the spanish translations. Hola, aquí están las traducciones en castellano.

# tweet 1

[blog] La Historia de censo2017, el Primer Paquete de rOpenSci Revisado en Castellano por @pachadotdev 

¡Acceder a los datos del censo de Chile :chile: nunca fue más fácil!

https://ropensci.org/blog/2021/06/27/censo2017/

#rstats

(1/2)

# tweet 2

¡Quizás lo mejor de todo es que este paquete se escribió Y revisó en Español para maximizar la accesibilidad!

¡agradecemos a los revisores de rOpenSci @​mapaulacaldas, @​fransvandunne y @​melvidoni por sus aportes al primer paquete de rOpenSci jamás escrito!

(2/2)

**Comment by steffilazerte** [comment]
Great! We're all done then, I'll push on Tuesday to publish and send out the tweets, and you can bask in the admiration :grin:

**Pull request merged**